stow.go

```diff
@@ -92,11 +94,16 @@ type Location interface {
 }
 
 type PresignRequestParams struct {
-	ExpiresIn   time.Duration
-	ContentMD5  string
-	ExtraParams map[string]interface{}
-	HttpMethod  HttpMethod
-	Metadata    map[string]string
```

Comment:
I thought we will keep Metadata... We can also keep "Metadata" :) is what I was trying to say... and only add `AddContentMD5Metadata bool`... but this is ok..